### DISCUSSION TITLE
[Question] How to solve the problem of robots jumping on one foot

### DISCUSSION THREAD

**Discussion by wangxudong1998** [fan-ziqi/robot_lab#61] [Q&A]
I trained my robot, and the result is that it walks by jumping on one foot or keeping its left foot in front and right foot in the back. How can I solve this problem?

**Comment by fan-ziqi** [comment]
You can tried to reduce the feet_air_time reward weight.

https://github.com/fan-ziqi/robot_lab/blob/713ed3e677a016df2ffc7374be2e1d0a42fd185a/exts/robot_lab/robot_lab/tasks/locomotion/velocity/config/fftai_gr1t1/flat_env_cfg.py#L15

**Comment by wangxudong1998** [comment]
After reducing feet_air_time reward weight, the robot I trained stopped in place (or took a long time to take a step). How can I solve this problem?

**Comment by Mewscot** [comment]
You may not only change feet_air_time reward, you should keep lin_vel_xy_exp.weights, feet_air_time weights and velocity ranges.lin_vel_x in "balance"
